Tests/FunctionalTests/Sources/FTRScrollViewTest.m

```diff
@@ -28,6 +28,10 @@ - (void)setUp {
   [self openTestViewNamed:@"Scroll Views"];
 }
 
+- (void)DISABLED_testFreezingScrollView {
```

Comment:
Why is this test disabled?

Reply:
I see. This is replicating an issue for us. Can you link the issue as well / create one if not already there?

Reply:
I don't think we should block this PR on that?

Feel free to create and link one to this Adarsh.